api/src/test/java/org/apache/iceberg/metrics/TestDefaultTimer.java

```diff
@@ -20,6 +20,7 @@
 
 import static java.util.concurrent.Executors.newFixedThreadPool;
 import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.apache.iceberg.TestHelpers.waitUntilAfter;
```

Comment:
changes on this file need to be reverted because we actually want to sleep in those tests